src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -51,6 +52,7 @@ object EntityClient {
   case class ImportBagit(workspaceNamespace: String, workspaceName: String, bagitRq: BagitImportRequest)
 
   case class ImportPFB(workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest, userInfo: UserInfo)
+  case class ImportPFBStatus(workspaceNamespace: String, workspaceName: String, jobId: String, userInfo: UserInfo)
```

Comment:
nit: can this be `PFBImportStatus`? i keep reading it as importing the status.

Reply:
done!